test/typewise-alert.test.js

```diff
@@ -1,6 +1,27 @@
-const alerts = require('../typewise-alert');
+const alerts =require('./../src/typeWiseAlert')
 const {expect} = require('chai');
+it('checkAndAlert', () => {
+expect(alerts.checkAndAlert('TO_CONTROLLER','PASSIVE_COOLING',-13));

```

Comment:
How do we verify if the right target has received the alert message? Test cases need to check that too...